doc/ug/getting_started_dv.md

```diff
@@ -88,3 +88,23 @@ Please refer to the [checklist]({{< relref "doc/project/checklist" >}}) to under
 
 The [UART DV](https://github.com/lowRISC/opentitan/tree/master/hw/ip/uart/dv) area can be used as a canonical example for making progress.
 If it is not clear on how to proceed, feel free to file an issue requesting assistance.
+
+## Reproduce DV failure in CI(private)
+
+Follow these steps to reproduce CI(private) failure
+
+1. Make sure the version of VCS is the same as the [one](https://github.com/lowRISC/opentitan-private-ci/blob/master/jobs.yml#L5) running in CI
+
+2. CI runs against an auto-generated merge commit, which effectively is the PR merge into master.
+This "merge" branch gets updated automatically by GitHub whenever you push to the PR branch, or when you close/re-open the PR.
+Get the extra database by doing the following (Assuming "upstream" is the name of your lowrisc/opentian repository)
+```console
+$ git fetch upstream pull/<PR_number>/merge:<your_branch>
```

Comment:
Perhaps `s/your_branch/temp_branch`?
For people like me who are (still :-/) confused by Git, also clarify here this has nothing to do with the branch that was originally pushed to open the PR.

Reply:
Maybe it would be better to split this differently?
```
git fetch upstream pull/<PR_number>/merge
git checkout -b <your_branch> FETCH_HEAD
```
I think that probably makes it clearer what's going on (and is what I do here!)